matron/src/device/device_monome.c

```diff
@@ -137,6 +174,11 @@ int dev_monome_grid_cols(struct dev_monome *md) {
     return monome_get_cols(md->m);
 }
 
+int dev_monome_arc_encs(struct dev_monome *md) {
+    // FIXME returning rows, libmonome currently doesn't support returning encoders
+    return monome_get_rows(md->m);
```

Comment:
if this is broken, consider returning a constant or removing the method completely.

Reply:
as a hack, scripts can get the number of encoders from the device name (which can be done in lua).

Reply:
that would be easier - is it considered a reliable enough way? not sure if it's different for DIY'd arcs. (the reason i didn't use a constant was because it would give me a compiler error about unused `md` variable, since warnings are treated as errors)

Reply:
right, just use `(void) md;` in the beginning of the function to avoid that warning. that's what we currently do to override that.

Reply:
cool, good to know!
going to remove this and move the implementation to lua as you suggested.